generators/app/templates/_dockerTaskGeneric.ps1

```diff
@@ -51,7 +51,13 @@ function CleanAll () {
 
 # Builds the Docker image.
 function BuildImage () {
-    $dockerFileName="Dockerfile.$Environment"
+    $dockerFileName = $Null
+    if ($Environment -eq "Release") {
+        $dockerFileName = "dockerfile"
+    }
+    else {
+        $dockerFileName = "dockerfile.$Environment"
```

Comment:
You could simplify this further by using release as the default and doing the if on if ($Environment -ne "Release") { as it was previously done below.

---
In reply to: [67450870](https://github.com/Microsoft/generator-docker/pull/67#discussion_r67450870) [](ancestors = 67450870)